experimental/auth-logic/templates/verifier_policy.auth.tmpl

```diff
@@ -0,0 +1,33 @@
+    // The verification policy trusts the endorsement file wrapper to give
```

Comment:
Changed it to just `Endorsement` instead of `EndorsementFile` everywhere under the assumption that that works. Let me know!

Reply:
> Doesn't it have to trust both?

I think about trust in terms of principals and delegations. And in that sense, I don't see the endorsement file as a principal. The product team that generates and signs the endorsement file is a principal that has to be trusted. The wrappers are also principals that should be trusted, IIUC. Since `"{{.AppName}}::Endorsement"` is on the left-hand side of `canSay` it should be a principal (right?). But it is not clear to me whether it represents the wrapper or the product team.

Reply:
I like thinking about trust in terms of principals and delegations! In the literature, a principal can be anything that can act (or take some action). This definition is extremely broad and can include: individual humans, groups of humans, machines, policies, software components, and other things (and principals have been used to represent all of these things and probably others in the literature). So I think any of these are valid uses of principals that we could consider: the endorsement wrapper, the endorsement file, or the group of humans that wrote specific claims in the endorsement file. It's up to us to decide how to split the claims into principals.

I agree with you that "the endorsement file" is not a useful principal because it's not a likely point of failure (and the name is changed in the code) -- the more "interesting principals" (and points of failure) are the other two: the endorsement file wrapper, and the group of humans that put claims in the endorsement file (the group being "the product team").

I think we don't yet have a good way to separate trust in the endorsement file wrapper and the product team making claims in the specific instance of the endorsement file wrapper. I have a few ideas about how to reduce trust in all the wrappers (and maybe allow us to separate trusting a wrapper and trusting the claims inside the thing being wrapped), and we can discuss this when we next meet.

I think this might make the most sense: the claims currently attributed to "Endorsement" really should be attributed to the product team (which is associated with a specific key). We can then assume for now that the wrapper is implicitly trusted to convert a claim given in one format (an endorsement file) into authorization logic. We can potentially improve on this later by making "trust in the wrapper" a part of our framework as well (potentially involving ideas to be discussed). 

Let me leave this as-is for now though, and open an issue to change this principal into a product team because I also was intending to go through the auth logic code to make sure cryptographic keys are used in ways that make sense, and I think that is related to this.

Reply:
Sounds good, and we can discuss more the next time we meet.